mesos-client/src/main/scala/com/mesosphere/mesos/client/MesosCalls.scala

```diff
@@ -25,7 +25,8 @@ class MesosCalls(frameworkId: FrameworkID) {
     * http://mesos.apache.org/documentation/latest/scheduler-http-api/#teardown
     */
   def newTeardown(): Call = {
-    Call.newBuilder()
+    Call
+      .newBuilder()
```

Comment:
I agree with most of the format changes... except this one..  why is this the preference?  can we fix it?   I really don't like it.

Reply:
I don't mind but it's not what we had and are used to.

Reply:
Ya that's weird